src/server/tweetFinder.js

```diff
@@ -205,3 +228,28 @@ const getTweetsAsResults = (data) => (
     (tweet) => ({ data: tweet.tweet, author: tweet.author, source: 'twitter' })
   )
 );
+
+export const getQuotedTweetFromParent = (req, res, id) => (
+  newPromiseChain()
+    .then(() => (
+       db.query(
+         makeTweetQuerySelectingFrom('TRAVERSE OUT FROM (SELECT OUT(\'QUOTED\') FROM (SELECT FROM Tweet WHERE id = :id))'),
+         {
+           params: {
+             id: req.params.id,
```

Comment:
any reason you're passing in the req and the params id then using req.params.id in getQuotedTweetFromParent